src/microbe_stage/systems/StrainSystem.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace Systems
+{
+    using Components;
+    using DefaultEcs;
+    using DefaultEcs.System;
+    using DefaultEcs.Threading;
+    using Godot;
+    using World = DefaultEcs.World;
+
+    [With(typeof(CompoundStorage))]
+    [With(typeof(OrganelleContainer))]
+    [With(typeof(MicrobeControl))]
+    [With(typeof(StrainAffected))]
+    [RunsAfter(typeof(PhysicsBodyDisablingSystem))]
+    public sealed class StrainSystem : AEntitySetSystem<float>
```

Comment:
Why doesn't this system rely on `IsUnderStrain`? It seems the purpose of that variable is so that the movement system sets that and then that greatly simplifies this system by not needing to act on so much microbe-specific information.